sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducer.cs

```diff
@@ -33,19 +33,21 @@ namespace Azure.Messaging.EventHubs
     ///
     public class EventHubProducer : IAsyncDisposable
     {
-        /// <summary>The maximum allowable size, in bytes, for a batch to be sent.</summary>
+        /// <summary>The minimum allowable size, in bytes, for a batch to be sent.</summary>
```

Comment:
I always thought this description was a bit confusing. The `MinimumBatchSizeLimit` is actually a "minimum limit", so I should still be able to send smaller sized batches, right?